docs/en/reference/_common_options.mdx

```diff
@@ -0,0 +1,69 @@
+|`--atime-mode=noatime` <VersionAdd>1.1</VersionAdd> |Control atime (last time the file was accessed) behavior, support the following modes:<br/><ul><li>`noatime` (default): set when the file is created or when `SetAttr` is explicitly called. Accessing and modifying the file will not affect atime, tracking atime comes at a performance cost, so this is the default behavior</li><li>`relatime`: update inode access times relative to mtime (last time when the file data was modified) or ctime (last time when file metadata was changed). Only update atime if atime was earlier than the current mtime or ctime, or the file's atime is more than 1 day old</li><li>`strictatime`: always update atime on access</li></ul>|
+|`--skip-dir-nlink=20` <VersionAdd>1.1</VersionAdd> |number of retries after which the update of directory nlink will be skipped (used for tkv only, 0 means never) (default: 20)|
+|`--skip-dir-mtime=100ms` <VersionAdd>1.2</VersionAdd>|skip updating attribute of a directory if the mtime difference is smaller than this value (default: 100ms)|
+
+#### Metadata cache related options {#metadata-cache-options}
+
+For metadata cache description and usage, refer to [Kernel metadata cache](../guide/cache.md#kernel-metadata-cache) and [Client memory metadata cache](../guide/cache.md#client-memory-metadata-cache).
+
+|Items|Description|
+|-|-|
+|`--attr-cache=1`|attributes cache timeout in seconds (default: 1), read [Kernel metadata cache](../guide/cache.md#kernel-metadata-cache)|
+|`--entry-cache=1`|file entry cache timeout in seconds (default: 1), read [Kernel metadata cache](../guide/cache.md#kernel-metadata-cache)|
+|`--dir-entry-cache=1`|dir entry cache timeout in seconds (default: 1), read [Kernel metadata cache](../guide/cache.md#kernel-metadata-cache)|
+|`--open-cache=0`|open file cache timeout in seconds (0 means disable this feature) (default: 0)|
+|`--open-cache-limit value` <VersionAdd>1.1</VersionAdd> |max number of open files to cache (soft limit, 0 means unlimited) (default: 10000)|
+
+#### Data storage related options {#data-storage-options}
+
+|Items|Description|
+|-|-|
+|`--storage=file`|Object storage type (e.g. `s3`, `gs`, `oss`, `cos`) (default: `"file"`, refer to [documentation](../reference/how_to_set_up_object_storage.md#supported-object-storage) for all supported object storage types).|
+|`--bucket=value`|customized endpoint to access object storage|
+|`--storage-class value` <VersionAdd>1.1</VersionAdd> |the storage class for data written by current client|
+|`--get-timeout=60`|the max number of seconds to download an object (default: 60)|
+|`--put-timeout=60`|the max number of seconds to upload an object (default: 60)|
+|`--io-retries=10`|number of retries after network failure (default: 10)|
```

Comment:
Would be great to add a hint about what happens after the max number of `io-retries` has been reached.